internal/storage/integration/fixtures/queries.json

```diff
@@ -23,7 +23,7 @@
     "Query": {
       "ServiceName": "query02-service",
       "OperationName": "",
-      "Tags": {
+      "Attributes": {
```

Comment:
@yurishkuro this cannot be parsed into a `pcommon.Map`. Any thoughts on what we should do here?